UnrealPlugin/Source/PlayFabGSDK/Public/MaintenanceSchedule.h

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include "MaintenanceSchedule.generated.h"
+
+USTRUCT(BlueprintType)
+struct FMaintenanceEvent
+{
+	GENERATED_BODY()
+public:
```

Comment:
True. Can't remember if this was generated this way in UE or what. It's in line with the rest of our structs though, so I'm going to leave it in case UE does want it there.